po/fr.po

```diff
@@ -1223,16 +1219,16 @@ msgstr "Une erreur est survenue lors du paramétrage du tube"
 msgid "An error occurred while redirecting file '%s'"
 msgstr "Une erreur est survenue pendant la redirection du fichier '%ls'"
 
-#: src/exec.cpp:714
-#, c-format
-msgid "Unknown function '%ls'"
-msgstr "Fonction inconnue '%ls'"
-
-#: src/exec.cpp:801
+#: src/exec.cpp:458
 #, c-format
 msgid "Unknown input redirection type %d"
 msgstr "Type de redirection d'entrée inconnu: %d"
 
+#: src/exec.cpp:827
+#, c-format
+msgid "Unknown function '%ls'"
+msgstr "Fonction inconnue '%ls'"
```

Comment:
Utilisez « Fonction '%ls' inconnue », c’est du meilleur français et ça sonne moins _traduction littérale_.

Use "Fonction '%ls' inconnue", that's better French and sounds less _litteral translation_.